docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,233 @@
+Very early on we created a shared configuration for our team with some custom policies, here are a few things we decided to do:
+
+- Group PRs for `minor` and `patch` dependencies
+- Internal dependencies could create a PR anytime of the day
+- Third party dependencies could create PRs only during the weekend
+
+This helped a lot to reduce the noise in PRs, the second month we got 400 Pull Requests, and on the third month only 200.
+
+## What we learned from automating dependencies updates
+
+- You need to be confident that your code coverage will warn you in case of updates. At the beginning we missed quite a few breaking updates because the build was green but the application broke as soon as it was deployed
+- Once you’re confident enough, auto-merge is a must have. In our team we enabled Renovate on about 100 of our repositories and generally spend 1–2 hours per week to stay up-to-date
+- “On the bleeding edge it’s not the edge that’s bleeding; it’s you”. When updating to a new major version as soon as it’s released you might encounter some surprises. It happened to us a few times that a patch version breaks the library. Usually a fix came out the next day, but we still spent a few hours debugging why the update broke our applications. We’ve opened quite a few Issues and sent some PRs to fix issues like this
+- Updating dependencies is one thing, but when should you release them? As our team mostly provides libraries, we don’t want to release them on every dependency upgrade (as this would create PRs downstream and create noise for them). We’ve decided to release right after critical upgrades or contributions and a dashboard informs us when a repository hasn’t been released for 30 days
+
+### A word on Renovate
+
+> The next section could look like an ad or a sponsored post; it’s not. We’re just big fans of the product.
+
+The ease of getting started with Renovate’s Docker image is what got us onboard easily.
+But what confirmed our choice is the insane crazy amount of features and [configuration options](https://docs.renovatebot.com/configuration-options/).
+
+Some features and options we enjoy:
+
+- [Shared configurations (presets)](https://docs.renovatebot.com/key-concepts/presets/), we have a Swissquote default configuration we set for all repositories, each team can extend it with their own practices
+- [Integration with GitHub’s Dependabot alerts](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) to raise the priority and send security remediation PRs as soon as possible
+- Each rule can be customized either globally [or specified per package](https://docs.renovatebot.com/configuration-options/#packagerules)
+- Works with your [private package registry](https://docs.renovatebot.com/getting-started/private-packages/)
+- Supports dozens of [languages and package managers](https://docs.renovatebot.com/modules/manager/#supported-managers): Maven, Docker, NPM, Docker Compose, Python
```

Comment:
```suggestion
- Supports more than 70 [languages and package managers](https://docs.renovatebot.com/modules/manager/#supported-managers): Maven, Docker, npm, Docker Compose, Python
```

Our docs page you've linked to has a list of about 74 "managers". A dozen means a group of twelve things, but how many is "dozens" exactly?

- 24
- 36
- 48
- 60
- 72 <--- this is closest to the actual number
- 84
- And so on

Changes:

- Replace `dozens of` with `more than 70`.
- Replace `NPM` with `npm`, as `npm` themselves don't use all-caps for their name

Reply:
This makes more sense indeed, I generally use a "dozen" to mean "more than ten" or "a lot" in some contexts.

But with your explanation, indeed, it doesn't make much sense to use its plural form